lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -607,4 +607,10 @@ def max_thread_limit
   rescue Gem::LoadError
     puts "No gettext-setup gem found, skipping gettext initialization" if Rake.verbose == true
   end
+  namespace :module do
+  desc "Runs all tasks to build a modules POT file for internationalization"
+    task :i18n do
```

Comment:
maybe we should call this `pot` instead, so if we have other tasks related to i18n, we won't have confusion.